salesforcecli/plugin-release-management
# Generate readme during cli PR build

### What does this PR do?
Generates a new readme prior to committing changes.

Sidenote: JIT plugins are not included in the readme. I think this is fine since they are not "shipped" with the CLI. 

### QA
- Pull branch
- cd to your local `cli` repo
- Run `../plugin-release-management/bin/dev cli:release:build --start-from-github-ref main --release-channel nightly --build-only`
- Confirm README has changes
- Confirm links to source work
  - Also verify on newly updated plugins
    - Check the `package.json`
    - Find a plugin that updated. Example `"@salesforce/plugin-deploy-retrieve": "1.18.2",`
    - Confirm links to source for that plugin contain the correct version (`1.18.2`)
 
### What issues does this PR fix or reference?
[@W-14197578@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-14197578)
